requirements.d/test.txt

```diff
@@ -1,3 +1,2 @@
 # the set of requirements necessary to run tests
-pytest>=2.6.3,<2.7
-fabric
+pytest
```

Comment:
I like to pin the pytest version because someone might already have pytest installed, just one that is too old and pip won't upgrade it automatically. Also, pytest occasionally makes breaking updates and tests no longer run. This has happened several times in the past (not necessarily with this project).  Current version is 2.6.4 but this is known to work with 2.6.3. Of course no idea if it works with future 2.7 or not but technically 2.6.* should not introduce any breaking changes so we know we're safe with that line until 2.7+ comes out.